### PR TITLE
Add recipe for adam-robotics Python package

### PR DESCRIPTION
**Pull request by traversaro**
Add recipe for the adam-robotics. The name of the Python module is `adam`, while the `PyPI` package is `adam-robotics`, so here for consistency with PyPI we use `adam-robotics`.

Repo: https://github.com/ami-iit/ADAM .
PyPI page: https://pypi.org/project/adam-robotics/  . 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by traversaro** [comment]
Blocked by https://github.com/conda-forge/staged-recipes/pull/21870 . 

@Giulero I added you as a mantainer, are you interested in being a mantainer? Please reply explicitly in the issue as it is required by conda-forge rules, thanks!

**Comment by Giulero** [comment]
Hi @traversaro ! Sure! Thanks a lot :)

**Comment by traversaro** [comment]
@Giulero the `test_gravity_term` test is not passing on Linux for both jax and pytorch, do you have any idea what could be the cause? Thanks!

**Comment by traversaro** [comment]
> @Giulero the `test_gravity_term` test is not passing on Linux for both jax and pytorch, do you have any idea what could be the cause? Thanks!

The `jaxlib`, `jax` and `pytorch` version are exactly the one that are in https://github.com/ami-iit/ADAM/pull/38 were tests are passing.

**Comment by traversaro** [comment]
> > @Giulero the `test_gravity_term` test is not passing on Linux for both jax and pytorch, do you have any idea what could be the cause? Thanks!
> 
> The `jaxlib`, `jax` and `pytorch` version are exactly the one that are in [ami-iit/ADAM#38](https://github.com/ami-iit/ADAM/pull/38) were tests are passing.

Ok, I was able to reproduce the issue. It is happening if one uses the `quay.io/condaforge/linux-anvil-cos7-x86_64` docker image, and not if one uses Ubuntu 22.04 .

**Comment by traversaro** [comment]
> > > @Giulero the `test_gravity_term` test is not passing on Linux for both jax and pytorch, do you have any idea what could be the cause? Thanks!
> > 
> > 
> > The `jaxlib`, `jax` and `pytorch` version are exactly the one that are in [ami-iit/ADAM#38](https://github.com/ami-iit/ADAM/pull/38) were tests are passing.
> 
> Ok, I was able to reproduce the issue. It is happening if one uses the `quay.io/condaforge/linux-anvil-cos7-x86_64` docker image, and not if one uses Ubuntu 22.04 in [ami-iit/ADAM#38](https://github.com/ami-iit/ADAM/pull/38) .

It turns out that the problem happens for glibc <= 2.25 and does not happen for glibc >= 2.26 : https://github.com/ami-iit/ADAM/pull/38#issuecomment-1546909762 . This is probably related to some strange interaction of jax/pytorch with glibc <= 2.25, and not directly related to adam-robotics. Furthermore, this happens just in a specific test case for some dependencies that by default are not insalled by jax, so I think we can just suppress the failing tests, and mark the PR as ready for review.

**Comment by traversaro** [comment]
@conda-forge/help-python @conda-forge/staged-recipes the PR is ready for review, thanks!

**Comment by traversaro** [comment]
Apparently there is still a test failure, sorry for the noise.

**Comment by traversaro** [comment]
> Apparently there is still a test failure, sorry for the noise.

The test failure is: 
~~~
=========================== short test summary info ============================
FAILED tests/test_Jax_computations.py::test_mass_matrix - assert array([[ 7.14...7924698e-10]]) == 0.0 ± 1.0e-05
  comparison failed
  Obtained: [[ 7.14008790e-06  7.82816528e-07  4.58020025e-07  2.08015834e-07\n   4.87857750e-08  8.21953381e-08  4.69735567e-08 -1.13577023e-08\n   2.54099177e-07  1.19819927e-07  1.03100486e-07 -3.15392471e-08\n  -1.04645499e-08  3.91728589e-08 -1.10762412e-07  4.05699531e-08\n   3.92313086e-09  8.21401824e-09 -2.56527078e-07  5.27741643e-09\n  -3.46103083e-08 -5.87602984e-09  4.24625441e-09  1.48892413e-08\n  -7.66407529e-08 -1.10474885e-09  3.15650091e-08 -2.69047599e-09\n   3.42780620e-10]\n [ 9.18137685e-07  7.14008789e-06  4.71265251e-07  7.04235150e-08\n   3.37433939e-08 -8.86945849e-07 -2.83939216e-07  6.54288461e-07\n  -3.76124935e-07  6.79950233e-08 -8.55753886e-08 -1.64003502e-08\n  -4.01479811e-08  7.45616458e-08 -2.44841447e-08 -1.91194266e-08\n   1.40608385e-08  6.03174155e-09  8.83806239e-09  2.14319464e-09\n   1.49834917e-09  1.02546856e-09  9.41354657e-10 -3.74468317e-10\n   2.94993377e-08  2.52040470e-08 -1.53446489e-09 -1.33263664e-09\n   3.10265614e-09]\n [ 3.30305538e-07  7.48943290e-08  1.09547852e-05 -1.19448241e-07\n   8.86945849e-07 -2.20025839e-07 -4.78000651e-07  3.89425058e-07\n  -1.72815839e-08  9.05419955e-08 -2.40888906e-09 -2.71317135e-08\n   3.33577070e-08  4.55870313e-08 -4.99442857e-08  2.64353144e-10\n   3.88213166e-08 -4.45887859e-08 -4.57126506e-08  1.10787905e-08\n   2.84535442e-08  2.81607280e-09 -3.05682079e-09  1.50096251e-07\n   2.74474311e-08  5.44393103e-09  2.38463004e-08 -2.78529439e-09\n   8.32440221e-10]\n [-1.52609854e-08 -2.87204354e-07  2.95633709e-08  1.60869381e-07\n  -5.30724781e-08  6.53581719e-08  1.83754468e-07 -7.94217573e-08\n   1.07679327e-07  5.04971190e-09  3.69307211e-08 -6.60592606e-09\n   2.01160257e-08 -1.56690204e-08  2.57803280e-08  5.57262711e-09\n   1.07366713e-08 -2.31222005e-08 -1.27067987e-08 -3.76017060e-09\n   5.75257404e-09  6.81488970e-10 -2.33507639e-09 -8.18989498e-09\n   8.35355859e-10 -2.82371931e-09 -1.16862816e-08 -1.04680179e-09\n   4.15762158e-10]\n [ 1.67995064e-07  1.41080903e-07  6.48527270e-07 -5.81947524e-08\n   3.21960819e-07  3.61396564e-08 -1.61436980e-07  7.17300300e-08\n   4.24002334e-08  6.27171778e-08  3.78266999e-08 -2.01088965e-08\n   1.00241820e-08  2.13294704e-08 -6.02748115e-08  1.31861109e-08\n  -3.80970019e-10  1.35087647e-08  3.26316812e-08 -7.15902266e-09\n   1.14885856e-08  8.97687025e-10 -9.58827911e-10  2.74622150e-08\n   1.64113988e-08 -1.08593377e-08  9.32365532e-10 -8.81165571e-10\n  -1.50896479e-10]\n [ 1.93954047e-07 -8.86945849e-07 -1.85484282e-07  5.97702365e-08\n   4.24185825e-09  3.19596535e-07  3.02748000e-08 -1.81647738e-07\n   1.20737473e-07 -5.39399512e-08  4.31581353e-08  2.03235348e-08\n   1.90250185e-08 -3.37300545e-08 -8.87981687e-09 -1.49202121e-08\n  -2.53692093e-10  5.33882084e-09  6.35159782e-08 -6.38519263e-09\n   9.34441699e-09 -4.81306160e-10 -1.40230651e-09 -6.24911742e-09\n  -2.10103562e-08 -3.33941855e-09 -5.42758891e-09 -3.21374358e-10\n  -7.69690041e-11]\n [ 4.69735567e-08 -2.83939216e-07 -4.78000651e-07  1.83754468e-07\n  -1.61436980e-07  3.02748000e-08  2.02510158e-07 -9.44755594e-08\n   1.07802505e-07 -1.57376757e-08  1.83868360e-08  1.00553346e-09\n   1.26576874e-08 -2.13601500e-08  4.98919188e-08  2.12376119e-09\n   1.18402713e-08  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-1.13577023e-08  6.54288462e-07  3.89425058e-07 -7.94217573e-08\n   7.17300300e-08 -1.81647738e-07 -9.44755594e-08  2.25844363e-07\n  -6.99633290e-08  7.59959212e-08 -2.25149778e-08 -2.74947233e-08\n  -1.03449829e-08  3.91476383e-08 -1.65139300e-08  1.92474287e-08\n   2.39908383e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 2.54099177e-07 -3.76124935e-07 -1.72815839e-08  1.07679327e-07\n   4.24002334e-08  1.20737473e-07  1.07802505e-07 -6.99633290e-08\n   1.87180680e-07  6.97922972e-09  5.24282712e-08 -4.98333813e-09\n   2.25625685e-08 -2.04208429e-08 -2.86745393e-10  5.70989175e-09\n   8.42744107e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 1.19819927e-07  6.79950233e-08  9.05419955e-08  5.04971190e-09\n   6.27171778e-08 -5.39399512e-08 -1.57376757e-08  7.59959212e-08\n   6.97922972e-09  7.49324356e-08  7.30929393e-09 -2.80203523e-08\n  -7.98703986e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 1.03100486e-07 -8.55753886e-08 -2.40888906e-09  3.69307211e-08\n   3.78266999e-08  4.31581353e-08  1.83868360e-08 -2.25149778e-08\n   5.24282712e-08  7.30929393e-09  3.80897440e-08 -1.31977325e-09\n   1.19891888e-08  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-3.15392471e-08 -1.64003502e-08 -2.71317135e-08 -6.60592606e-09\n  -2.01088965e-08  2.03235348e-08  1.00553346e-09 -2.74947233e-08\n  -4.98333813e-09 -2.80203523e-08 -1.31977325e-09  1.33424190e-08\n   3.81365272e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-1.04645499e-08 -4.01479811e-08  3.33577070e-08  2.01160257e-08\n   1.00241820e-08  1.90250185e-08  1.26576874e-08 -1.03449829e-08\n   2.25625685e-08 -7.98703986e-09  1.19891888e-08  3.81365272e-09\n   2.38362106e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 3.91728589e-08  7.45616458e-08  4.55870313e-08 -1.56690204e-08\n   2.13294704e-08 -3.37300545e-08 -2.13601500e-08  3.91476383e-08\n  -2.04208429e-08  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  4.85006195e-08 -1.24278575e-08  3.08180189e-08\n  -3.43326845e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-1.10762412e-07 -2.44841447e-08 -4.99442857e-08  2.57803279e-08\n  -6.02748115e-08 -8.87981687e-09  4.98919188e-08 -1.65139300e-08\n  -2.86745393e-10  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00 -1.24278575e-08  2.53019713e-08 -9.34122255e-10\n   5.18863331e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 4.05699531e-08 -1.91194266e-08  2.64353144e-10  5.57262711e-09\n   1.31861109e-08 -1.49202121e-08  2.12376119e-09  1.92474287e-08\n   5.70989175e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  3.08180189e-08 -9.34122255e-10  1.04473678e-08\n  -1.86707111e-10  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 3.92313086e-09  1.40608385e-08  3.88213166e-08  1.07366713e-08\n  -3.80970019e-10 -2.53692095e-10  1.18402713e-08  2.39908383e-09\n   8.42744107e-09  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00 -3.43326845e-09  5.18863331e-09 -1.86707111e-10\n   4.24189024e-10  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 8.21401824e-09  6.03174155e-09 -4.45887859e-08 -2.31222005e-08\n   1.35087647e-08  5.33882084e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  2.26486540e-08  2.33105053e-09  4.99897215e-09\n  -4.01415539e-09 -1.79463807e-10  8.17907258e-10  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-2.56527078e-07  8.83806239e-09 -4.57126506e-08 -1.27067987e-08\n   3.26316812e-08  6.35159783e-08  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  2.33105053e-09  2.05019353e-08 -2.80852192e-09\n   3.28776562e-10 -5.14730599e-10 -7.45648577e-10  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 5.27741643e-09  2.14319464e-09  1.10787905e-08 -3.76017060e-09\n  -7.15902266e-09 -6.38519263e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  4.99897215e-09 -2.80852192e-09 -1.13676549e-09\n  -3.01435185e-10  1.93480044e-10 -5.55179598e-10  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-3.46103083e-08  1.49834914e-09  2.84535442e-08  5.75257404e-09\n   1.14885856e-08  9.34441699e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00 -4.01415539e-09  3.28776562e-10 -3.01435185e-10\n  -8.09961972e-10 -9.78617048e-10 -3.39158301e-10  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [-5.87602984e-09  1.02546856e-09  2.81607280e-09  6.81488970e-10\n   8.97687025e-10 -4.81306160e-10  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00 -1.79463807e-10 -5.14730599e-10  1.93480044e-10\n  -9.78617048e-10  8.95835732e-10 -6.50398295e-12  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 4.24625441e-09  9.41354657e-10 -3.05682079e-09 -2.33507639e-09\n  -9.58827911e-10 -1.40230651e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  8.17907258e-10 -7.45648577e-10 -5.55179598e-10\n  -3.39158301e-10 -6.50398295e-12 -7.03023310e-10  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00]\n [ 1.48892413e-08 -3.74468317e-10  1.50096251e-07 -8.18989498e-09\n   2.74622150e-08 -6.24911742e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  2.06132534e-08\n   6.81891610e-09 -5.48565656e-09  6.89114887e-09  7.37903225e-10\n  -1.01097619e-10]\n [-7.66407529e-08  2.94993377e-08  2.74474311e-08  8.35355861e-10\n   1.64113988e-08 -2.10103562e-08  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  6.81891610e-09\n   1.65032324e-09 -1.96961850e-09  1.10972913e-09 -1.86773955e-10\n  -8.99017073e-10]\n [-1.10474890e-09  2.52040470e-08  5.44393103e-09 -2.82371931e-09\n  -1.08593377e-08 -3.33941855e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00 -5.48565656e-09\n  -1.96961850e-09  4.34521860e-09 -6.58059744e-10  3.99349884e-10\n   6.58120791e-10]\n [ 3.15650091e-08 -1.53446489e-09  2.38463005e-08 -1.16862816e-08\n   9.32365534e-10 -5.42758891e-09  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  6.89114887e-09\n   1.10972913e-09 -6.58059744e-10  5.09959242e-10  7.51962017e-10\n   1.85187843e-10]\n [-2.69047599e-09 -1.33263664e-09 -2.78529438e-09 -1.04680179e-09\n  -8.81165571e-10 -3.21374358e-10  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  7.37903225e-10\n  -1.86773955e-10  3.99349884e-10  7.51962017e-10 -6.46550712e-10\n   1.19370721e-10]\n [ 3.42780620e-10  3.10265614e-09  8.32440221e-10  4.15762158e-10\n  -1.50896479e-10 -7.69690024e-11  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00  0.00000000e+00\n   0.00000000e+00  0.00000000e+00  0.00000000e+00 -1.01097619e-10\n  -8.99017073e-10  6.58120791e-10  1.85187843e-10  1.19370721e-10\n  -1.77924698e-10]]
  Expected: 0.0 ± 1.0e-05
FAILED tests/test_Jax_computations.py::test_CMM - assert array([[ 7.14...2716048e-10]]) == 0.0 ± 1.0e-05
  comparison failed
  Obtained: [[ 7.14008790e-06  7.82816528e-07  4.58020025e-07  2.08015834e-07\n   4.87857750e-08  8.21953381e-08  4.69735567e-08 -1.13577023e-08\n   2.54099177e-07  9.74681848e-08  7.32981641e-08 -3.15392471e-08\n  -6.73925961e-09  3.26536009e-08 -1.25663573e-07  4.05699531e-08\n   3.92313086e-09  8.21401824e-09 -2.56527078e-07  5.27741643e-09\n  -3.46103083e-08 -2.15073954e-09  4.24625441e-09  1.48892413e-08\n  -1.70361081e-08 -1.10474885e-09  2.03891382e-08 -8.27830839e-10\n   3.42780620e-10]\n [ 9.18137685e-07  7.14008789e-06  4.71265251e-07  7.04235150e-08\n   3.37433939e-08 -8.86945849e-07 -2.83939216e-07  6.54288461e-07\n  -3.76124935e-07  5.30938621e-08 -8.55753886e-08 -1.26750599e-08\n  -3.26974005e-08  7.45616458e-08 -5.05611768e-08 -2.28447169e-08\n   1.95364503e-09  6.03174155e-09  8.83806239e-09  5.86848493e-09\n   1.49834917e-09  1.95679114e-09  2.80399981e-09 -3.74468317e-10\n   1.45981765e-08  4.01052082e-08 -1.53446489e-09 -1.33263664e-09\n   1.24001099e-09]\n [ 3.30305538e-07  7.48943290e-08  1.09547852e-05 -1.19448241e-07\n   8.86945849e-07 -2.20025839e-07 -4.78000651e-07  3.89425058e-07\n  -1.72815839e-08  8.30914149e-08  1.17937802e-08 -2.52690683e-08\n   3.33577070e-08  4.55870313e-08 -6.48454469e-08  3.98964344e-09\n   3.13707360e-08 -4.45887859e-08 -4.57126506e-08  3.62820990e-09\n   2.10029636e-08  3.74739538e-09 -3.05682079e-09  1.50096251e-07\n   3.48980117e-08  5.44393103e-09  6.10992034e-08 -9.22649238e-10\n  -9.88823537e-11]\n [ 2.15071196e-07 -1.65103528e-07  2.03578261e-08  2.26493308e-07\n  -9.09578577e-08  5.79044207e-08  1.75315490e-07 -7.67623302e-08\n   1.46542683e-07  2.40923197e-09  3.21633650e-08 -6.10797810e-09\n   1.47937445e-08 -2.05298604e-08  2.01850244e-08  9.62261759e-09\n   1.45230274e-08 -3.01948781e-08 -2.40802968e-08 -1.91437680e-09\n   4.54377936e-09  3.86584458e-10 -2.65028957e-09  7.69838679e-10\n  -7.09656919e-09 -2.13602538e-09  7.22980268e-10 -2.64500193e-11\n   1.06097044e-09]\n [ 1.72749934e-07  1.22256211e-07 -1.57670259e-07 -3.41471807e-08\n   3.28525452e-07  3.28776038e-08 -8.12796851e-08  4.58096447e-08\n   6.85506676e-08  6.14575868e-08  3.27338271e-08 -2.38717031e-08\n   2.50246942e-09  1.95942034e-08 -7.42617527e-08  1.70851393e-08\n  -4.87148820e-09  1.18940865e-08  3.63916921e-08 -4.39694174e-09\n   9.85216223e-09  2.71984667e-10 -1.66924272e-09  2.56386200e-08\n   7.28263122e-09 -5.18513863e-09 -6.26062219e-10 -6.96664105e-10\n  -7.80755564e-11]\n [ 1.74951594e-07  5.84244453e-08 -3.38166586e-07  5.79044209e-08\n  -3.75536658e-08  2.76114952e-07  1.64478246e-08 -1.38912631e-07\n   9.21059715e-08 -4.40959193e-08  4.01122355e-08  3.11061768e-08\n   1.42544145e-08 -3.16089930e-08 -8.53690955e-09 -1.43471104e-08\n   1.57871440e-09  1.02525217e-08  3.04004034e-08 -4.97635216e-09\n   7.55852758e-09  3.65798213e-10 -7.99140103e-10 -8.55565327e-09\n   5.60276861e-09  1.69827756e-09 -3.79675543e-09 -5.54192821e-10\n   1.92716048e-10]]
  Expected: 0.0 ± 1.0e-05
=========== 2 failed, 39 passed, 4 deselected, 22 warnings in 25.48s ===========
~~~

**Comment by traversaro** [comment]
The flaky tests were fixed by backporting https://github.com/ami-iit/ADAM/pull/40 , now the recipe is actually ready for review, thanks! @conda-forge/help-python @conda-forge/staged-recipes

File: recipes/adam-robotics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ami-iit/ADAM/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4a726ac0c0ce6735eb51be30f512583d63721a270d16cb74738f78615540a6cb
+  patches:
+    - 40.patch
```
Comment (ocefpaf) [inline recipes/adam-robotics/meta.yaml:12]: Can you add a comment here explaining the patch and/or if it was submitted upstream?

Comment (traversaro) [reply recipes/adam-robotics/meta.yaml:14]: Yes, done at https://github.com/conda-forge/staged-recipes/pull/21894/commits/ed5d32a332a63a88f1b19fbd8f055385eba26241 , thanks!

**Pull request merged**